ci/README.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+Travis integration is prepared to check pull request with different tests. It uses docker images as clean environment with all build dependencies onboard. 
+Such an approach reduces time because it uses pull command instead of compiling dependencies and does not require "clean" operations. Depending on 
+current Orion test logic, unit and functional testing should be executed separately, so travis build is divided into stages. 'Optional' test
+does not depend on 3rd party tools and does not use binaries of Orion, so it can be executed directly. 
+
+## Supported tests
```

Comment:
I'd suggest to describe test in the order they are executed by Travis. Something like this:

> Currently the following test stages are executed (in this order):
> 
> * Compliance, including:
>   * style check
>   * payload check
>   * file compliance check
> * Unit test
> * Functional test

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/85f9adb0d02f1a9d1906bea2198af60546f9b612